frontend/routes/auth/transactions/details/transaction-metadata.tsx

```diff
@@ -88,6 +89,30 @@ export const TransactionMetadata = ({ transaction }: TransactionSectionPropertie
       <div key="transaction-details-sent" data-testid={locators.transactionMetadataSent}>
         {formatDateTime(new Date(transaction.receivedAt).getTime())}
       </div>
+    ],
+    [
+      'Automatically Confirmed',
+      <div
+        key="transaction-details-automatically-confirmed"
+        data-testid={locators.transactionMetadataAutomaticallyConfirmed}
+      >
+        <Tooltip
+          description={
+            <span>
+              The auto consent setting can be found and changed under{' '}
+              <NavLink
+                className="text-white underline"
+                to={`${FULL_ROUTES.connections}/${encodeURIComponent(transaction.origin)}`}
+              >
+                Connections
+              </NavLink>
+              .
+            </span>
+          }
+        >
+          <span>{transaction.autoApproved === undefined ? '-' : transaction.autoApproved ? '👍' : '👎'}</span>
```

Comment:
**comment (non-blocking)**: I did wonder where the nested ternary was going to come in. It's manageable.